web/modules/custom/collection/collection.module

```diff
@@ -254,3 +254,25 @@ function collection_search_api_query_search_api_field_field_collection_content_a
   // Remove the collection itself from the results.
   $query->addCondition($entity->getEntityType()->getKey('id'), $entity->id(), '<>');
 }
+
+/**
+ * Implements hook_form_alter().
+ *
+ * Remove non-collection related actions (adding/removing roles)
+ * from member overview page.
+ */
+function collection_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  // Bulk operation form ids vary on entity id.
+  if (strstr($form_id, 'views_form_og_members_overview_default') === FALSE) {
```

Comment:
Sadly, we can't. The form id is something like views_form_og_members_overview_default_rdf_entity_http://joinup....
So it contains the id of the group entity.